source/Calamari.Tests/Fixtures/StructuredVariables/JsonFormatVariableReplacerFixture.cs

```diff
@@ -126,6 +126,17 @@ public void ShouldReplaceWholeObject()
                         TestEnvironment.AssentJsonDeepCompareConfiguration);
         }
 
+        [Test]
```

Comment:
Not strictly related to this changeset, but we discovered that this test case was missing while implementing similar behaviour for XML.